kimchi/src/verifier_index.rs

```diff
@@ -238,13 +239,13 @@ where
             endo: self.cs.endo,
             lookup_index,
             linearization: self.linearization.clone(),
-            fr_sponge_params: self.cs.fr_sponge_params.clone(),
+            fr_sponge_params: G::sponge_params().clone(),
             fq_sponge_params: self.fq_sponge_params.clone(),
         }
     }
 }
 
-impl<G: CommitmentCurve> VerifierIndex<G>
+impl<G: CommitmentCurve + KimchiCurve> VerifierIndex<G>
```

Comment:
since `KimchiCurve: CommitmentCurve` you can just have `G: KimchiCurve` here right?

Reply:
indeed